Jenkinsfile

```diff
@@ -36,7 +36,7 @@ devToolsProject.run(
       groovylint: { groovylint.checkSingleFile(path: './Jenkinsfile') },
       molecule: {
         withEnv(["ANSIBLE_ROLES_PATH=${data.rolesPath}"]) {
-          data.venv.run('molecule --debug test')
+          data.venv.run('molecule --debug test --all')
```

Comment:
Because I added another scenario. :smile: 

Without any flags, `molecule test` tests the `default` scenario (ie, `<project root>/molecule/default/molecule.yml`. You need to pass `--all` to tell molecule to test everything (ie, `<project root>/molecule/*/molecule.yml`